doc/source/release-notes.rst

```diff
@@ -2,6 +2,19 @@
 Release History
 ***************
 
+v1.0.0 (2019-06-03)
```

Comment:
should be 0.4.0

Reply:
@ke-zhang-rd I think you marked this as resolved but you did not comment. Per today's discussion, I think we now agree that this should be 0.4.0.

Reply:
Yes, I updated on cantrips but forgot here, good catch